events_api.go

```diff
@@ -168,6 +171,22 @@ func (a *EventsAPIServer) handleEvent(innerEvent slackevents.EventsAPIInnerEvent
 }
 
 func (a *EventsAPIServer) handleMessageEvent(ev *slackevents.MessageEvent) {
+	edited := &slack.Edited{}
```

Comment:
Ah yeah I see, I introduced a nil pointer panic when it is not set :+1: 

However maybe it would be better to leave it at `nil` actually unless it is set in the given event?
```suggestion
	var edited *slack.Edited
```